src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -231,7 +233,17 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
             {
                 var symbol = symbols.First().OriginalDefinition;
 
-                var documentation = symbol.GetDocumentationParts(semanticModel, position, formatter, cancellationToken);
+                // if generating quick info for an attribute, bind to the class instead of the constructor
+                if (token.GetAncestors(t => IsAttributeSyntax(t)).Any())
+                {
+                    var attributeType = semanticModel.Compilation.GetTypeByMetadataName("System.Attribute");
```

Comment:
Might as well use ITypeSymbolExtensions.IsAttribute, since that's good enough everywhere else.